.gitignore

```diff
@@ -2,12 +2,20 @@
 *.o
 *.pb.cc
 *.pb.h
+*~
```

Comment:
This should probably be going in people’s ~/.config/git/ignore, not individual repositories’ .gitignores.

Reply:
*~ is a fairly common POSIX/UNIX convention, so I'd personally prefer to keep it in the project's .gitignore. I agree with the general comment that editor-specific environment stuff should go in per-user configuration.